libraries/shared/src/PathUtils.h

```diff
@@ -58,7 +58,7 @@ class PathUtils : public QObject, public Dependency {
     static QString generateTemporaryDir();
     static bool deleteMyTemporaryDir(QString dirName);
 
-    static int removeTemporaryApplicationDirs(QString appName = QString::null);
+    static int removeTemporaryApplicationDirs(QString appName = "");
```

Comment:
```suggestion
    static int removeTemporaryApplicationDirs(QString appName = QString());
```
Implementation of `removeTemporaryApplicationDirs(…)` tests `.isNull()`.

Reply:
Gah. Good catch, fixing both.

Maybe we should just test for isEmpty instead?

Reply:
Possibly. Though would need to check program logic to ensure there aren't any unintended consequences. So probably easier just to keep using null values per current code.